code/WorkInProgress/recycling/crusher.dm

```diff
@@ -127,3 +127,18 @@
 	..()
 	if(status & (NOPOWER|BROKEN))	return
 	use_power(500)
+
+/obj/machinery/crusher/New()
+	..()
+	var/counter = 0
+	var/turf/T = get_turf(src)
+	for(var/obj/thing in T)
```

Comment:
Why not just check T.contents.len

Reply:
Could also make it simpler by doing:

```
var/obj/machinery/crusher/C = locate(/obj/machinery/crusher) in T
if (C)
    //output text, del object

```

Reply:
Well might as well write out the whole thing:

```
if (T.contents.len > 100)
    //output failure and del
else
    var/obj/machinery/crusher/C = locate(/obj/machinery/crusher) in T
    if (C)
        //output text, del object
```